Gems/ROS2/Code/Source/SdfAssetBuilder/SdfAssetBuilderSettings.h

```diff
@@ -34,5 +34,7 @@ namespace ROS2
         bool m_useArticulations = true;
         // By default, fixed joint in URDF files that are processed by libsdformat are preserved
         bool m_urdfPreserveFixedJoints = true;
+        // When true, .dae/.stl mesh files are imported into the project folder to allow the AP to process them
```

Comment:
Nit pick. Without this parameter we will also trigger AP to rebuild asset. The difference is that we instead copying, try to search file.